plutus-core/docs/BuiltinsOverview.md

```diff
@@ -0,0 +1,456 @@
+## Preface
+
+This document elaborates on the implementation of the builtins machinery.
+
+Before you proceed, make sure that you understand the high-level concepts at play, for that we have the a [separate](./BuiltinsIntro.md) doc.
+
+## Built-in functions: overview
+
+### Module alignment
+
+The module alignment of the built-in functions machinery (sans most of costing, which we'll include in a separate section):
+
+<pre>
```

Comment:
FYI: checkout <https://asciiflow.com> , maybe it can be of help next time

Reply:
That's cool, thanks. I'll probably do it by hand still, but now for no good reason.